src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -764,7 +764,7 @@ export const Deposit: React.FC<DepositProps> = ({
         expectedAmountOutThorBaseUnit,
         amountCryptoBaseUnit,
         asset,
-        apy: opportunityData?.apy,
+        apy: opportunityData?.apy ?? '0',
```

Comment:
I added another early return but TS is drunk because we already early return upper in [a3d2662](https://github.com/shapeshift/web/pull/7373/commits/a3d2662b1500407616b930af9442d372c5391216)